pkgs/applications/networking/aws-client-vpn/default.nix

```diff
@@ -62,17 +63,19 @@ stdenv.mkDerivation rec {
         done < <(find $prefix -name '*.so' -maxdepth 1 -print0)
       }
 
-      patchPrefix $out/awsvpnclient "AWS VPN Client" createdump
-      patchPrefix $out/awsvpnclient/Service ACVC.GTK.Service createdump
+      patchPrefix $out/lib/awsvpnclient "AWS VPN Client" createdump
+      patchPrefix $out/lib/awsvpnclient/Service ACVC.GTK.Service createdump
 
-      for prog in $out/awsvpnclient/"AWS VPN Client" $out/awsvpnclient/Service/ACVC.GTK.Service; do
+      for prog in $out/lib/awsvpnclient/"AWS VPN Client" $out/lib/awsvpnclient/Service/ACVC.GTK.Service; do
         wrapProgram "$prog" \
           --set LD_PRELOAD "${libredirect}/lib/libredirect.so" \
-          --set NIX_REDIRECTS /opt/awsvpnclient=$out/awsvpnclient \
+          --set NIX_REDIRECTS /opt/awsvpnclient=$out/lib/awsvpnclient \
           "''${gappsWrapperArgs[@]}"
       done
     '';
 
+  dontStrip = true;
```

Comment:
Why?

Reply:
Without this I get
```
Failed to create CoreCLR, HRESULT: 0x80004005
```
when trying to run the binary (I found this workaround mentioned [here](https://github.com/actions/runner/issues/918#issuecomment-774140470)). Admittedly I don't really understand the issue or why this fixes it; would be good to know if there's a better solution.

Reply:
This seems to happen with all dotnet applications, no idea why.
IMHO just slap a comment to explain setting this, as in https://github.com/NixOS/nixpkgs/pull/114162/files#diff-07dd1ac705a7596aa88012cb4ed0854081fcb79530bfd88b06b5510c7fe0c571R24-R26

Reply:
Good call, added a note in baef5fe